addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -387,19 +387,23 @@
         <field name="tag">hr_attendance_greeting_message</field>
     </record>
 
-    <record id="hr_attendance_action_install_kiosk_pwa" model="ir.actions.client">
```

Comment:
Would keep the old one unused, with a comment telling it is outdated. That way we don't try to remove stuff in stable, which is always a bad idea.